resources/repos.json

```diff
@@ -970,10 +970,10 @@
         },
         "dependencies": {
             "Minecraft Version(s)": {
-                "1": "1.14.x, 1.15.x, 1.16.x"
+                "1": "1.14.x, 1.15.x, 1.16.x, 1.17.x"
             },
             "Slimefun Version": {
-                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#18\">RC-18</a>"
+                "1": "<a class=\"link_info\" href=\"/builds/TheBusyBiscuit/Slimefun4/stable/#18\">RC-28</a>"
             }
```

Comment:
These versions are not supported by build no. 1 though, so that needs to be changed.